calicoctl/commands/node/status.go

```diff
@@ -79,21 +79,44 @@ Description:
 				os.Exit(1)
 			}
 
-			// Connect to the bird socket and get the data if calico-node container is running
-			c, err := net.Dial("unix", "/var/run/calico/bird.ctl")
-			if err != nil {
-				log.Fatalf("Error connecting to the BIRD socket: %v", err)
+			versions := []string{"4", "6"}
+
+			// Loop over and try getting info about both IP versions
+			for _, v := range versions {
+				birdSuffix := ""
+				ipSep := "."
+				if v == "6" {
+					birdSuffix = "6"
+					ipSep = ":"
+				}
+				fmt.Printf("\nIPv%s BGP status", v)
+
+				// Connect to the bird socket and get the data
+				c, err := net.Dial("unix", fmt.Sprintf("/var/run/calico/bird%s.ctl", birdSuffix))
+				if err != nil {
+					log.Printf("Error connecting to BIRD socket: %v", err)
+					continue
+				}
+				defer c.Close()
+
+				fmt.Println()
```

Comment:
Because previous print is not always printf, it's table render when there is no error.